### PR TITLE
SLE-681: Integration tests for marking issues (#497)

### PR DESCRIPTION
**Pull request by thahnen**
Marking an issue in connected mode: Trying to resolve an issue with insufficient permissions and then with administrator rights. Now on Cirrus CI!

### CODE REVIEW

**Comment by thahnen** [comment]
Hi @henryju, got everything to run. Tried to not use a second orchestrator but for some reason it only works this way, I suspect that it does not clean up perfectly after every different test, as running the "Mark issue as ..." test before all the others (when having them in the same class sharing the orchestrator) works.
But we are not losing that much time with a second orchestrator as I would have expected.

File: its/src/org/sonarlint/eclipse/its/SonarQubeMarkAsTest.java [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+        .useDefaultAdminCredentialsForBuilds(true)
+        .keepBundledPlugins()
+        .setSonarVersion(System.getProperty("sonar.runtimeVersion", "LATEST_RELEASE[8.9]"))
+        // Ensure SSE are processed correctly just after SQ startup
+        .setServerProperty("sonar.pushevents.polling.initial.delay", "2")
+        .setServerProperty("sonar.pushevents.polling.period", "1")
+        .setServerProperty("sonar.pushevents.polling.last.timestamp", "1")
+        .restoreProfileAtStartup(
+          URLLocation.create(FileLocator.toFileURL(FileLocator.find(FrameworkUtil.getBundle(SonarQubeConnectedModeTest.class), new Path("res/java-sonarlint.xml"), null))))
+        // Ensure SSE are processed correctly just after SQ startup
```
[the remark this answers is not in the document]
Comment (thahnen) [reply its/src/org/sonarlint/eclipse/its/SonarQubeMarkAsTest.java:72]: @henryju I just copied it from the old test, are you sure it wasn't done on purpose?

File: its/src/org/sonarlint/eclipse/its/SonarQubeConnectedModeTest.java [hunk 2]
```diff
@@ -267,7 +281,101 @@ public void shouldAutomaticallyUpdateRuleSetWhenChangedOnServer() throws Excepti
...
+      .build());
+
+    // 5) Remove user rights from project
+    adminWsClient.permissions().removeGroup(new RemoveGroupWsRequest()
+      .setProjectKey(MAVEN2_PROJECT_KEY)
+      .setGroupName("sonar-users")
+      .setPermission("issueadmin"));
+
+    // 6) Remove connections and reconnect with user
+    cleanBindings();
```
[the remark this answers is not in the document]
Comment (thahnen) [reply its/src/org/sonarlint/eclipse/its/SonarQubeConnectedModeTest.java]: Done

File: its/src/org/sonarlint/eclipse/its/SonarQubeMarkAsTest.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (henryju) [reply its/src/org/sonarlint/eclipse/its/SonarQubeMarkAsTest.java:72]: Probably the other test needs to be fixed as well ;)

[the remark this answers is not in the document]
Comment (thahnen) [reply its/src/org/sonarlint/eclipse/its/SonarQubeMarkAsTest.java]: Done!

**Pull request merged**